### PR TITLE
HARP-5631 Fixing missing tiles during tilting / rotating view

### PR DESCRIPTION
**Pull request by nzjony**
Signed-off-by: Jonathan Stichbury <2533428+nzjony@users.noreply.github.com>

### CODE REVIEW

**Comment by nzjony** [comment]
@heremaps-bot retest this please

File: @here/harp-mapview/lib/VisibleTileSet.ts [hunk 1]
```diff
@@ -572,17 +572,40 @@ export class VisibleTileSet {
...
             worldCenter.z
         );
         const worldLeftGeoPoint = this.options.projection.unprojectPoint(worldLeftPoint);
-        const offsetRange = Math.ceil(
-            Math.abs((worldGeoPoint.longitude - worldLeftGeoPoint.longitude) / 360)
+        // We multiply by Sqrt(2) because we need to account for a rotated view (in which case there
+        // are more tiles that can be seen).
+        const offsetRange = MathUtils.clamp(
+            Math.ceil(
+                Math.abs((worldGeoPoint.longitude - worldLeftGeoPoint.longitude) / 360) * 1.414
```
Comment (haraldF) [inline @here/harp-mapview/lib/VisibleTileSet.ts]: 1.414 looks quite like a magic number... In C++, we have `M_SQRT2`, would it make sense to have something like a `MathUtils.Sqrt2` constant? Or, if we don't need it anywhere else, a `const Sqrt2 = 1.414;` ?

Comment (nzjony) [reply @here/harp-mapview/lib/VisibleTileSet.ts]: I did a search for 1.414 and didn't find it used elsewhere so i have added a local const.

File: @here/harp-mapview/lib/VisibleTileSet.ts [hunk 2]
```diff
@@ -572,17 +572,41 @@ export class VisibleTileSet {
...
             worldCenter.x - worldLengthHorizontal,
             worldCenter.y,
             worldCenter.z
         );
         const worldLeftGeoPoint = this.options.projection.unprojectPoint(worldLeftPoint);
-        const offsetRange = Math.ceil(
-            Math.abs((worldGeoPoint.longitude - worldLeftGeoPoint.longitude) / 360)
+        // We multiply by Sqrt(2) because we need to account for a rotated view (in which case there
+        // are more tiles that can be seen).
+        const Sqrt2 = 1.414;
```
Comment (okanishchev) [inline @here/harp-mapview/lib/VisibleTileSet.ts]: You can use Math.SQRT2 here

Comment (nzjony) [reply @here/harp-mapview/lib/VisibleTileSet.ts]: Beautiful.

**Pull request merged**